pkg/backend/snapshot.go

```diff
@@ -271,7 +272,7 @@ func (ssm *sameSnapshotMutation) mustWrite(step *deploy.SameStep) bool {
 		sortDeps := func(deps []resource.URN) []resource.URN {
 			result := make([]resource.URN, len(deps))
 			copy(result, deps)
-			sort.Slice(result, func(i, j int) bool { return deps[i] < deps[j] })
+			slices.Sort(result)
```

Comment:
Is this because `sort.Slice` isn't stable and `slices.Sort` is?

Reply:
Oh, I see `result` vs `deps`.

Reply:
I just did an audit of all the other uses of `sort.Slice` in the repo, and there aren't any other bugs like this. 👍